src/containers/Home/Home.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Home = () => (
+  <div className='members'>
```

Comment:
Move the css to `src/containers/Home/styles.css`

Reply:
Done.